packages/landing-gatsby/src/templates/processTamplate.js

```diff
@@ -0,0 +1,342 @@
+                  </Tab>
+
+                  <Tab onClick={handleClick} active={active === 1} id={1}>
+                    <span role="img" aria-label="Policies">
+                      📜
+                    </span>{' '}
+                    Policies
+                  </Tab>
+                  <Tab onClick={handleClick} active={active === 2} id={2}>
+                    <span role="img" aria-label="Roles">
+                      🎭
+                    </span>{' '}
+                    Roles
+                  </Tab>
+                  <Tab onClick={handleClick} active={active === 3} id={3}>
+                    <span role="img" aria-label="Tools">
+                      ⚒️
+                    </span>{' '}
+                    Tools
+                  </Tab>
+                  <Tab onClick={handleClick} active={active === 4} id={4}>
+                    <span role="img" aria-label="Values">
+                      💖
+                    </span>{' '}
+                    Values
+                  </Tab>
+                </Tabs>
+                <>
+                  <Fragment>
+                    <ContentWrapper active={active === 0}>
+                      <p>
+                        Organisational structures, formats and style of
+                        governance that is needed to adapt the process."
+                      </p>
+                      {structures.map((structure) => (
+                        <ExternalLink href={structure.link}>
+                          <ContentHover>
+                            <Heading content={structure.name} />
+                            <MDReactComponent text={structure.description} />
+                          </ContentHover>
+                        </ExternalLink>
+                      ))}
+                    </ContentWrapper>
+                    <ContentWrapper active={active === 1}>
+                      <p>
+                        Policies and frameworks used in this process. These can
+                        be individualized based on the organisation’s need and
+                        background.
+                      </p>
+                      {policies.map((policy) => (
+                        <ContentScroll>
+                          <Heading content={policy.name} />
+                          <MDReactComponent text={policy.description} />
+                        </ContentScroll>
+                      ))}
+                    </ContentWrapper>
+                    <ContentWrapper active={active === 2}>
+                      <p>
+                        Mandatory roles that are affected or created for this
+                        process to work properly.{' '}
+                      </p>
+                      {roles.map((role) => (
+                        <ContentScrollSmall>
+                          <Heading content={role.title} />
+                          <Text content={role.purpose} />
+                          <Text content={role.domains} />
```

Comment:
Warning: Failed prop type: Invalid prop `content` of type `array` supplied to `Text`, expected `string`.